### DISCUSSION TITLE
funtion skip

### DISCUSSION THREAD

**Discussion by saeidn95** [vladmandic/human#438] [Q&A]
I like to know more about the basis for the method used in function skip https://github.com/vladmandic/human/blob/62396317f5c31b2ec5bc7de2e9eb53b8978b9f28/src/image/image.ts#L313 

I assume it is used to skip processing frames if there is not enough changes between two frames. What is the basis for this method? Is there a reference for this?

**Comment by vladmandic** [comment]
what is the basis for this method? its not accurate, but its fast and cheap and can be done with almost no overhead.
and since skip frames is a utility item to speed up processing, it doesn't need to be precise.

**Reply by saeidn95** [reply]
Good!, and what us skipFrame=99 here?  https://github.com/vladmandic/human/blob/62396317f5c31b2ec5bc7de2e9eb53b8978b9f28/src/config.ts#L386C7-L386C21

Is it to set the number of frame to skip?

**Reply by vladmandic** [reply]
but the comment in the exact link you've pasted documents it?
its upper limit of frames that are allowed to skip IF difference is below threshold (if difference is above threshold, then they will not be skipped anyhow).
